### PR TITLE
Features/gitlab source

### PR DESCRIPTION
**Pull request by koffeinfrei**
Adds gitlab as a source to the git statistics.

Only feature missing from gitlab is language statistics, which isn't covered by the gitlab api (-> https://github.com/panter/dashboard.panter.ch/issues/20).

Ready for review.

### CODE REVIEW

**Comment by koffeinfrei** [comment]
ping @huerlisi / @toupeira

File: lib/gitlab_client.rb [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+  module Notes
+    def merge_request_notes(project, merge_request, options={})
+      get("/projects/#{project}/merge_requests/#{merge_request}/notes", :query => options)
+    end
+  end
+end
+
+class GitlabClient
+  def initialize
+    Gitlab.endpoint = 'https://git.panter.ch/api/v3'
```
Comment (toupeira) [inline lib/gitlab_client.rb]: I would add this to `.env` as well in the interest of keeping the code company-agnostic ;-)

**Comment by toupeira** [comment]
Looks good I think :+1:

File: lib/gitlab_client.rb (hunk 1 above)
Comment (koffeinfrei) [inline lib/gitlab_client.rb]: :+1:

**Pull request merged**